packages/core/src/account/simple.ts

```diff
@@ -186,6 +203,33 @@ export async function createSimpleSmartAccount<
 
   return {
     ...base,
-    getSigner: () => simpleAccount.getSigner() as TSigner,
+    getSigner: () => simpleAccount.getSigner(),
   };
 }
+
+// Example usage:
+//
+// const a = await createSimpleSmartAccount({
+//   chain: sepolia,
+//   signer: LocalAccountSigner.mnemonicToAccountSigner("OWNER_MNEMONIC"),
+//   factoryAddress: getDefaultSimpleAccountFactoryAddress(sepolia),
+//   transport: http(),
+// });
+//
+// const b = await createSimpleSmartAccount({
+//   chain: sepolia,
+//   signer: LocalAccountSigner.mnemonicToAccountSigner("OWNER_MNEMONIC"),
+//   factoryAddress: getDefaultSimpleAccountFactoryAddress(sepolia),
+//   transport: http(),
+//   entryPoint: getEntryPoint(sepolia),
+// });
+//
+// const c = await createSimpleSmartAccount({
+//   chain: sepolia,
+//   signer: LocalAccountSigner.mnemonicToAccountSigner("OWNER_MNEMONIC"),
+//   factoryAddress: getDefaultSimpleAccountFactoryAddress(sepolia),
+//   transport: http(),
+//   entryPoint: getEntryPoint(sepolia, { version: "0.7.0" }),
+// });
+// const ep = c.getEntryPoint()
+// type d = typeof ep.version // "0.7.0"
```

Comment:
nit: remove this or make this a `jsdoc` comment and use `@example` in the comment to include this code

Reply:
done